corehq/apps/reports/forms.py

```diff
@@ -130,9 +144,10 @@ def __init__(self, display_privacy_disclaimer, *args, **kwargs):
                 'send_to_owner',
                 'attach_excel',
                 'recipient_emails',
+                'language',
                 crispy.HTML(
                     render_to_string('reports/partials/privacy_disclaimer.html')
-                ) if display_privacy_disclaimer else None
```

Comment:
My intention with this line was to prevent displaying the privacy disclaimer if there are no UCRs (non-UCR reports do not have privacy concerns that I am aware of)